docs/testing.md

```diff
@@ -197,4 +197,95 @@ val response1 = basicRequest.get(uri"http://api.internal/a").send()
 
 val response2 = basicRequest.post(uri"http://api.internal/b").send()
 // response2 will be whatever a "real" network call to api.internal/b returns
+```
+
+## Testing WebSockets
+
+Stub methods `whenRequestMatches` and `whenAnyRequest` also allow to specify behavior for `openWebsocket` call when used together with `thenRespondWebSocket` or `thenHandleOpenWebSocket`. `thenRespondWebSocket` is intended for cases when sttp supplied
```

Comment:
Unneeded line break at the end of this line (i.e. lines 204 and 205 should be one line)

Reply:
:+1: I've edited my changes to apply line breaks before 121 column. Markdown should render single white space in same way, regardless it's line break or space.

Reply:
Excellent, I didn't know that